python/cudf/cudf/pandas/fast_slow_proxy.py

```diff
@@ -881,6 +881,12 @@ def _assert_fast_slow_eq(left, right):
         assert_eq(left, right)
 
 
+class CudfPandasFallbackError(Exception):
```

Comment:
Suggestion: I could prefer `FallbackError` to make this naming a little more generic.

Reply:
I would like `ProxyFallbackError` better.